ros/src/sensing/drivers/camera/packages/vectacam/package.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package>
+  <name>vectacam</name>
+  <version>0.0.0</version>
+  <description>The vectacam package</description>
+
+  <maintainer email="autoware@googlegroups.com">nu-amc</maintainer>
+
+  <license>GPL</license>
```

Comment:
`GPL` ?  License comment in `vectacam/nodes/vectacam/vectacam_node.cpp` is 3-BSD.